### PR TITLE
DOCS-2565: Add point cloud support to realsense

### PR DESCRIPTION
**Pull request by sguequierre**
* Update based on template updates
* Add more information about API endpoints supported

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,9 +57,28 @@ The following attributes are available for `viam:camera:realsense` cameras:
...
+
+Before testing the module, make sure that your machine is connected to the Viam app, displaying as **Live** in the part status dropdown in the top right corner of the machine's page.
+
+### Camera API
 
-If you do not want to use the Viam Registry, for Linux distros, download the module AppImage from our servers to your robot.
+Once the `realsense` model is configured on your machine, you can access the depth and color data the camera streams through the [Viam camera API](https://docs.viam.com/components/camera/#api).
+The following methods of the Viam camera API are supported:
+
+- [`GetPointCloud`](https://docs.viam.com/components/camera/#getpointcloud)
```
Comment (sguequierre) [inline README.md:69]: @hexbabe would it be correct to say that `GetPointCloud` is depth, `GetImage` color and `GetImages` both and should I add that info here if so?

File: README.md [hunk 2]
```diff
@@ -29,15 +22,15 @@ On the new component panel, copy and paste the following attribute template into
...
+> For more information, see [Configure a Machine](https://docs.viam.com/manage/configuration/).
 
 ## Attributes
 
 The following attributes are available for `viam:camera:realsense` cameras:
 
 | Name | Type | Inclusion | Description |
 | ---- | ---- | --------- | ----------- |
-| `sensors` | list | **Required** | The RealSense data streams you want your robot to sense from. A list that contain the strings `color` and/or `depth`. The sensor that comes first in the list is designated the "main sensor", and is the image that gets returned by `get_image` calls and appears in the **Control** tab on the [Viam app](https://app.viam.com). If you would like a list of images from all listed sensors simultaneously, use `get_images`.  |
+| `sensors` | list | **Required** | The RealSense data streams you want your robot to sense from. A list that contain the strings `color` and/or `depth`. The sensor that comes first in the list is designated the "main sensor", and is the image that gets returned by `get_image` calls and appears in the **Control** tab on the [Viam app](https://). If you would like a list of images from all listed sensors simultaneously, use `get_images`.  |
```
Comment (sguequierre) [inline README.md:33]:
```suggestion
| `sensors` | list | **Required** | The RealSense data streams you want your robot to sense from. A list that contain the strings `color` and/or `depth`. The sensor that comes first in the list is designated the "main sensor", and is the image that gets returned by `get_image` calls and appears in the **CONTROL** tab on the [Viam app](https://app.viam.com). If you would like a list of images from all listed sensors simultaneously, use `GetImages()`.  |
```

File: README.md [hunk 3]
```diff
@@ -64,9 +57,28 @@ The following attributes are available for `viam:camera:realsense` cameras:
...
+
+### Camera API
 
-If you do not want to use the Viam Registry, for Linux distros, download the module AppImage from our servers to your robot.
+Once the `realsense` model is configured on your machine, you can access the depth and color data the camera streams through the [Viam camera API](https://docs.viam.com/components/camera/#api).
+The following methods of the Viam camera API are supported:
+
+- [`GetPointCloud`](https://docs.viam.com/components/camera/#getpointcloud)
+- [`GetImage`](https://docs.viam.com/components/camera/#getimage)
+- [`GetImages`](https://docs.viam.com/components/camera/#getimage)
```
Comment (sguequierre) [inline README.md:71]:
```suggestion
- [`GetImages`](https://docs.viam.com/components/camera/#getimages)
```

File: README.md (hunk 1 above)
Comment (hexbabe) [reply README.md:69]: Hmm, sometimes `GetPointCloud` returns a point cloud with color data. Looks kinda like my slack profile pic. So point clouds are always depth and sometimes color.

Screenshot 2024-06-18 at 11 37 21 AM

**Review by hexbabe — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sguequierre) [reply README.md:69]: Woah

Comment (sguequierre) [reply README.md:69]: Thank you!

Comment (sguequierre) [inline README.md:69]:
```suggestion
- [`GetPointCloud`](https://docs.viam.com/components/camera/#getpointcloud): always depth and sometimes color
```

File: README.md [hunk 4]
```diff
@@ -64,9 +57,28 @@ The following attributes are available for `viam:camera:realsense` cameras:
...
+Before testing the module, make sure that your machine is connected to the Viam app, displaying as **Live** in the part status dropdown in the top right corner of the machine's page.
+
+### Camera API
 
-If you do not want to use the Viam Registry, for Linux distros, download the module AppImage from our servers to your robot.
+Once the `realsense` model is configured on your machine, you can access the depth and color data the camera streams through the [Viam camera API](https://docs.viam.com/components/camera/#api).
+The following methods of the Viam camera API are supported:
+
+- [`GetPointCloud`](https://docs.viam.com/components/camera/#getpointcloud)
+- [`GetImage`](https://docs.viam.com/components/camera/#getimage)
```
Comment (sguequierre) [inline README.md:70]:
```suggestion
- [`GetImage`](https://docs.viam.com/components/camera/#getimage): color
```

File: README.md [hunk 5]
```diff
@@ -64,9 +57,28 @@ The following attributes are available for `viam:camera:realsense` cameras:
...
+- [`GetImages`](https://docs.viam.com/components/camera/#getimages)
```
Comment (sguequierre) [inline README.md:71]:
```suggestion
- [`GetImages`](https://docs.viam.com/components/camera/#getimages): both depth and color
```

File: README.md (hunk 1 above)
Comment (hexbabe) [reply README.md]: Np!

File: README.md [hunk 6]
```diff
@@ -1,21 +1,14 @@
-# Intel RealSense Modular Component
+# [`realsense` module](https://app.viam.com/module/viam/realsense)
 
-This is a [Viam module](https://docs.viam.com/manage/configuration/#modules) for the [Intel® RealSense™](https://github.com/IntelRealSense/librealsense) family of cameras. Registered at https://app.viam.com/module/viam/realsense.
+This [module](https://docs.viam.com/registry/#modular-resources) implements the [`rdk:component:camera` API](https://docs.viam.com/components/camera/) in an `viam:camera:realsense` model.
+Configure this model on your machine to stream image and depth data from the [Intel® RealSense™](https://github.com/IntelRealSense/librealsense) family of cameras to Viam.
```
Comment (andf-viam) [inline README.md:4]: Nit: can probably skip the trademarks

```suggestion
Configure this model on your machine to stream image and depth data from the [Intel RealSense](https://github.com/IntelRealSense/librealsense) family of cameras to Viam.
```

File: README.md [hunk 7]
```diff
@@ -1,21 +1,14 @@
-# Intel RealSense Modular Component
+# [`realsense` module](https://app.viam.com/module/viam/realsense)
 
-This is a [Viam module](https://docs.viam.com/manage/configuration/#modules) for the [Intel® RealSense™](https://github.com/IntelRealSense/librealsense) family of cameras. Registered at https://app.viam.com/module/viam/realsense.
+This [module](https://docs.viam.com/registry/#modular-resources) implements the [`rdk:component:camera` API](https://docs.viam.com/components/camera/) in an `viam:camera:realsense` model.
```
Comment (andf-viam) [inline README.md:3]:
```suggestion
This [module](https://docs.viam.com/registry/#modular-resources) implements the [`rdk:component:camera` API](https://docs.viam.com/components/camera/) in a `viam:camera:realsense` model.
```

File: README.md [hunk 8]
```diff
@@ -64,9 +57,28 @@ The following attributes are available for `viam:camera:realsense` cameras:
...
+### Camera API
 
-If you do not want to use the Viam Registry, for Linux distros, download the module AppImage from our servers to your robot.
+Once the `realsense` model is configured on your machine, you can access the depth and color data the camera streams through the [Viam camera API](https://docs.viam.com/components/camera/#api).
+The following methods of the Viam camera API are supported:
+
+- [`GetPointCloud`](https://docs.viam.com/components/camera/#getpointcloud): always depth and sometimes color
+- [`GetImage`](https://docs.viam.com/components/camera/#getimage): color
+- [`GetImages`](https://docs.viam.com/components/camera/#getimages): both depth and color
+- [`GetProperties`](https://docs.viam.com/components/camera/#getproperties)
```
Comment (andf-viam) [inline README.md:72]: Nit: if some descriptions provided, provide all. Little more context per as well:

```suggestion
- [`GetPointCloud`](https://docs.viam.com/components/camera/#getpointcloud): returns depth data and can return color data depending on the provided image
- [`GetImage`](https://docs.viam.com/components/camera/#getimage): returns color data
- [`GetImages`](https://docs.viam.com/components/camera/#getimages): returns both depth and color data
- [`GetProperties`](https://docs.viam.com/components/camera/#getproperties): returns intrinsic properties of a camera
```

File: README.md [hunk 9]
```diff
@@ -64,9 +57,28 @@ The following attributes are available for `viam:camera:realsense` cameras:
...
+
+You can view the data your camera streams live on the **CONTROL** tab of the Viam app.
+For more information, see [Control Machines](https://docs.viam.com/fleet/control/).
+
+### Locally install the module
+
+If you do not want to use the Viam registry, for Linux distros, download the module AppImage from our servers to your robot.
 
 ```
 sudo curl -o /usr/local/bin/viam-camera-realsense http://packages.viam.com/apps/camera-servers/viam-camera-realsense-latest-aarch64.AppImage
```
Comment (andf-viam) [inline README.md:84]: Adding a link to local module instructions:

````suggestion
If you are using a Linux machine, and do not want to use the Viam registry, you can download the module AppImage from our servers directly to your machine.

```
sudo curl -o /usr/local/bin/viam-camera-realsense http://packages.viam.com/apps/camera-servers/viam-camera-realsense-latest-aarch64.AppImage
```

Then, follow the instructions to [add a local module](https://docs.viam.com/registry/configure/#add-a-local-module) to add the local instance of the `realsense` module to your machine.
Provide an **Executable path** of `/usr/local/bin/viam-camera-realsense` when adding the module.

````

**Review by andf-viam — APPROVED** [review]
LGTM, just some tiny optional suggestions. Nice work!

**Pull request merged**